gulpfile.js

```diff
@@ -30,12 +31,18 @@ gulp.task('copySource', gulpTasks.copy({
   src: `${SOURCE_DIR}/guide/**/*.jsx`,
 }).task);
 
-gulp.task('deploy', gulpTasks.deploy({
-  accessKeyId: process.env.AWS_ACCESS_KEY_ID || 'No ENV',
-  bucketName: process.env.AWS_BUCKET_UI_FRAMEWORK || 'No ENV',
-  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY || 'No ENV',
-  src: `${DISTRIBUTION_DIR}/**/*.*`,
-}).task);
+gulp.task('deployToGitHubPages', () => (
+  gulp.src('./dist/**/*')
+    .pipe(ghPages())
+));
+
+gulp.task('deploy', done => {
+  runSequence(
+    'production',
+    'deployToGitHubPages',
+    done
+  );
+});
```

Comment:
But sometimes you don't want to run production first, e.g. when you want to deploy the unminified scripts or when you want to run production first, then do end-to-end tests and then deploy without running production again or when you want to deploy to AWS instead. You can always run `gulp production && gulp deployToGitHubPages` instead of having this task. We can keep it but I don't think it's that useful.